client-and-server/client/script.js

```diff
@@ -21,18 +21,33 @@ var updateQuantity = function(evt) {
     return;
   }
 
-  var isAdding = evt.target.id === "add";
+  var isAdding = evt && evt.target.id === "add";
   var inputEl = document.getElementById("quantity-input");
   var currentQuantity = parseInt(inputEl.value);
 
   document.getElementById("add").disabled = false;
   document.getElementById("subtract").disabled = false;
 
   // Calculate new quantity
-  var quantity = isAdding ? currentQuantity + 1 : currentQuantity - 1;
+  var quantity = evt
+    ? isAdding
+      ? currentQuantity + 1
+      : currentQuantity - 1
+    : currentQuantity;
 
   inputEl.value = quantity;
-  document.getElementById("total").textContent = quantity * 5;
+  var numberFormat = new Intl.NumberFormat(i18next.language, {
```

Comment:
Want to add some comments or move this into a translateButtonText function so it can be a bit clearer what's going on

Reply:
Added some comments and moved the basePrice into the .ENV file